### PR TITLE
MessageChannel Feature Detection

### PR DESCRIPTION
**Pull request by kongaraju**
MessageChannel API detection added to Modernizr tests

### CODE REVIEW

File: feature-detects/messagechannel.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "name": "W3C Reference",
+    "href": "https://www.w3.org/TR/2011/WD-webmessaging-20110317/#message-channels"
+  }, {
+    "name": "MDN documentation",
+    "href": "https://developer.mozilla.org/en-US/docs/Web/API/Channel_Messaging_API/Using_channel_messaging"
+  }]
+}
+!*/
+/* DOC
+Detects support for  `Message Channel` API from the Channel Messaging spec. Message Channels  provide a simple way for communication between threads.
```
Comment (patrickkettner) [inline feature-detects/messagechannel.js]: There is weird spacing, its a bit overly verbose, and not 100% accurate.

Could you update to something like

```
Detects support for Message Channels, a way to communicate between different browsing contexts (iframes, workers, etc)
```

**Comment by patrickkettner** [comment]
one comment, otherwise :+1:

could you update, then squish down to a single commit - thanks!

**Comment by kongaraju** [comment]
What to do mocha tests time out, I didn't find any error in test.

**Comment by patrickkettner** [comment]
that was a just a env failure. please squash down to a single commit and we will be good to go.

cheers!

**Comment by kongaraju** [comment]
is there anything stopping to ship it?

**Comment by patrickkettner** [comment]
hey @kongaraju!
So sorry for not replying sooner. I went ahead and rebased your code in #2026, merging with credit to you as the author - thanks again for the PR!

**Pull request closed without merging**